src/components/QuestDetails/QuestDetails.stories.tsx

```diff
@@ -450,3 +450,17 @@ export const WithExternalSyncButton: Story = {
     ]
   }
 }
+
+export const EndDateInFuture: Story = {
+  args: {
+    ...props,
+    endDate: '2025-04-10T00:00:00.000Z'
```

Comment:
can you move this like 20 years in the future? we are already past this date time so it shows as "ended on"